<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="But_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_0">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_dot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_equals">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_plus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_minus">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_multi">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_div">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_mod">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_close">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_open">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="But_x">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>705</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Переменная для графика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Значащая переменная</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>710</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Х=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_x">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>705</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="But_cos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_acos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_asin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_atan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_log">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_sin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_tan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_ln">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_sqrt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Х</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_pow">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>630</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранять последний результат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_credit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>840</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>КРЕДИТ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_vklad">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>840</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ВКЛАД</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_x_min">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>750</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>750</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Хmin=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_x_max">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>750</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>750</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Хmax=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>790</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ymin=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_y_min">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>790</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_y_max">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>790</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>790</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ymax=</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
